NuRadioReco/examples/cr_analysis/analyse_sim.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import os
+import glob
+import helper_cr_eff as hcr
+import json
+import NuRadioReco.modules.io.eventReader as eventReader
+from NuRadioReco.framework.parameters import stationParameters as stnp
+from NuRadioReco.framework.parameters import showerParameters as shp
+from NuRadioReco.utilities import units
+import argparse
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
```

Comment:
Can this example get a brief description (docstring) at the top to describe what it is doing?

Reply:
This would be on me. Will do.